cmd/kubeadm/app/cmd/init.go

```diff
@@ -290,6 +292,11 @@ func AddInitOtherFlags(flagSet *flag.FlagSet, initOptions *initOptions) {
 		&initOptions.skipCertificateKeyPrint, options.SkipCertificateKeyPrint, initOptions.skipCertificateKeyPrint,
 		"Don't print the key used to encrypt the control-plane certificates.",
 	)
+	flagSet.BoolVar(
+		&initOptions.printManifest, options.PrintManifest, initOptions.printManifest,
+		"Print the addon manifests to STDOUT instead of  install them.",
+	)
```

Comment:
i think we should not add the flag to the main `init` or `init phase addon`, but *only* to the addon phases:
- `init phase addon coredns --print-manifest`
- `init phase addon kube-proxy --print-manifest`

Reply:
Thanks. I will move the flag to the addon phases

Reply:
`Print the addon manifests to STDOUT instead of  install them.`
->
`Print the addon manifest to STDOUT instead of installing them.`

Reply:
The plural form is correct here (with s). We can adjust the message again once the rest of the code org is done.